### ISSUE TITLE
git needs Autoconf as a dep (at least on clean centos-7)

### ISSUE THREAD

**Issue by michaelkarlcoleman** [easybuilders/easybuild-easyconfigs#5738]
In a clean centos-7 image, latest git fails to build for lack of Autoconf.  This is probably true of all git versions.  

Adding Autoconf-2.69 as a dependency fixed this, though it's probably actually a build dependency.

```
Singularity centos-7-easybuild-3.5.1.simg:/projects/eb-racs> eb    --parallel=28    --robot    --robot-paths=/projects/eb-racs/easybuild-easyconfigs/easybuild/easyconfigs/    --prefix=/projects/eb-racs/p    --buildpath=/tmp/eb-racs       --trace    --debug    --disable-cleanup-builddir    --disable-cleanup-tmpdir --rebuild git-2
.14.1-GCCcore-6.4.0.eb
== temporary log file in case of crash /tmp/eb-FzB7KN/easybuild-bm3xxR.log
== resolving dependencies ...
== processing EasyBuild easyconfig /projects/eb-racs/easybuild-easyconfigs/easybuild/easyconfigs/g/git/git-2.14.1-GCCcore-6.4.0.eb
== building and installing git/2.14.1-GCCcore-6.4.0...
  >> installation prefix: /projects/eb-racs/p/software/git/2.14.1-GCCcore-6.4.0
== fetching files...
  >> sources:
  >> /projects/eb-racs/p/sources/g/git/v2.14.1.tar.gz [SHA256: ccc366d5d674fb755fd98d219c23f2b4e5da8a49d8582a6314813b280d75536b]
== creating build dir, resetting environment...
  >> build dir: /tmp/eb-racs/git/2.14.1/GCCcore-6.4.0
== unpacking...
  >> running command:
        [started at: 2018-01-26 11:33:54]
        [output logged in /tmp/eb-FzB7KN/easybuild-run_cmd-Qo8WD_.log]
        tar xzf /projects/eb-racs/p/sources/g/git/v2.14.1.tar.gz
  >> command completed: exit 0, ran in < 1s
== patching...
== preparing...
  >> loading toolchain module: GCCcore/6.4.0
  >> loading modules for build dependencies:
  >>  * binutils/2.28-GCCcore-6.4.0
  >> loading modules for (runtime) dependencies:
  >>  * cURL/7.55.1-GCCcore-6.4.0
  >>  * expat/2.2.4-GCCcore-6.4.0
  >>  * gettext/0.19.8.1-GCCcore-6.4.0
  >>  * Perl/5.26.0-GCCcore-6.4.0
  >> defining build environment for GCCcore/6.4.0 toolchain
== configuring...
  >> running command:
        [started at: 2018-01-26 11:33:55]
        [output logged in /tmp/eb-FzB7KN/easybuild-run_cmd-E4cn8a.log]
        make configure &&  ./configure --prefix=/projects/eb-racs/p/software/git/2.14.1-GCCcore-6.4.0 --with-perl=${EBROOTPERL}/bin/perl --enable-pthreads='-lpthread'
  >> command completed: exit 2, ran in < 1s
== FAILED: Installation ended unsuccessfully (build directory: /tmp/eb-racs/git/2.14.1/GCCcore-6.4.0): build failed (first 300 chars): cmd "make configure &&  ./configure --prefix=/projects/eb-racs/p/software/git/2.14.1-GCCcore-6.4.0 --with-perl=${EBROOTPERL}/bin/perl --enable-pthreads='-lpthread'" exited with exit code 2 and output:
GIT_VERSION = 2.14.1
    GEN configure
/bin/sh: autoconf: command not found
```

**Comment by Micket** [comment]
I included autoconf build dep in #6116 (I removed autoconf rpms from the system before building as a test)

**Comment by boegel** [comment]
this was fixed in #5718, and `Autotools` is included as a build dep in recent `git` easyconfigs, so closing...
